pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -14,6 +14,8 @@ The subgraph definition consists of a few files:
 
 - `AssemblyScript Mappings`: [AssemblyScript](https://github.com/AssemblyScript/assemblyscript) code that translates from the event data to the entities defined in your schema (e.g. `mapping.ts` in this tutorial)
 
+> In order to use your subgraph on The Graph's decentralized network, you will need to [create an API key](/deploying/subgraph-studio-faqs/#2-how-do-i-create-an-api-key) and [add signal](/network/curating/#how-to-signal) to your subgraph with at least [10,000 GRT](/network-transition-faq/#how-can-i-ensure-that-my-subgraph-will-be-picked-up-by-indexer-on-the-graph-network).
```

Comment:
- 10,000 GRT is based on empirical data from the current state of the network. 

- It may not be clear to the reader why we recommend this number. 

- Subgraph developers may read this in the future, when the state of the network is different. 

I'd be cautious whenever such numbers are communicated. A fixed number, communicated in the official docs, could set a precedence for how much signal a subgraph "should" have. This influence could make the market less free.

I would at least change this to a loose recommendation, and not "You will need to signal with at least 10,000 GRT"

Reply:
Good call! Agreed. Cleaned that up. Thank you!